core/src/test/scala/tech/beshu/ror/unit/acl/blocks/rules/FieldsRuleTests.scala

```diff
@@ -203,6 +203,18 @@ class FieldsRuleTests extends WordSpec with MockFactory with Inside {
         )
       }
     }
+    "request is ROR admin request" should {
+      "not match" in {
```

Comment:
I though that the problem was the fact that ROR request was not matched by one block with filter rule (was matched by the next block)? Please double check it

Reply:
resolved on slack